### PR TITLE
fix golint errors for pkg/master

### PR DESCRIPTION
**Pull request by m3ngyang**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
fix a wrong comment

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

**Comment by cheftako** [comment]
Thanks for helping with our quality.
I assume for looking at the changes you are trying to get our lint errors down.
If we don't clear all the errors in the directory and then mark the directory as clean then it is very easy for someone else to (re)introduce more lint errors in the directory.
Can you remove pkg/master from https://github.com/kubernetes/kubernetes/blob/master/hack/.golint_failures#L140?

**Comment by m3ngyang** [comment]
Do I need to fix all golint errors in `pkg/master` before removing it from `golint_failures`.  There exists some other golint errors, such as exported functions/structs lacking comments. @cheftako

**Comment by cheftako** [comment]
@m3ngyang Yes you would need to fix the remaining golint errors for pkg/master. However I think that is the right way to proceed.

**Comment by m3ngyang** [comment]
Done, I've fixed all go-lints errors for pkg/master. @cheftako

File: pkg/master/controller.go [hunk 1]
```diff
@@ -131,11 +131,13 @@ func (c *completedConfig) NewBootstrapController(legacyRESTStorage corerest.Lega
 	}
 }
 
+// PostStartHook only runs Start currently
```
Comment (cheftako) [inline pkg/master/controller.go]: PostStartHook initiates the core controller loops that must exist for bootstrapping.

File: pkg/master/controller.go [hunk 2]
```diff
@@ -131,11 +131,13 @@ func (c *completedConfig) NewBootstrapController(legacyRESTStorage corerest.Lega
...
 func (c *Controller) PostStartHook(hookContext genericapiserver.PostStartHookContext) error {
 	c.Start()
 	return nil
 }
 
+// PreShutdownHook only runs Stop currently
```
Comment (cheftako) [inline pkg/master/controller.go]: PreShutdownHook triggers the actions needed to shut down the API Server cleanly.

File: pkg/master/controller.go [hunk 3]
```diff
@@ -171,6 +173,7 @@ func (c *Controller) Start() {
 	c.runner.Start()
 }
 
+// Stop defines process of stopping the master
```
Comment (cheftako) [inline pkg/master/controller.go]: Stop cleans up this API Servers endpoint reconciliation leases so another master can take over more quickly.

File: pkg/master/controller.go (hunk 1 above)
Comment (m3ngyang) [reply pkg/master/controller.go]: thx for your replenishment. done.

File: pkg/master/controller.go (hunk 2 above)
Comment (m3ngyang) [reply pkg/master/controller.go]: done

File: pkg/master/controller.go (hunk 3 above)
Comment (m3ngyang) [reply pkg/master/controller.go]: done

**Comment by m3ngyang** [comment]
@cheftako all tests have completed.

**Comment by cheftako** [comment]
Hi @m3ngyang, you need to assign one of the "OWNERS" from the links above to get merge approval.

**Comment by m3ngyang** [comment]
/assign @liggitt 
/assign @deads2k 
PTAL~

File: pkg/master/master.go [hunk 4]
```diff
@@ -403,7 +405,7 @@ func (c completedConfig) New(delegationTarget genericapiserver.DelegationTarget)
 	return m, nil
 }
```
Comment (liggitt) [inline pkg/master/master.go]: Don't unexport this

File: pkg/master/master.go [hunk 5]
```diff
@@ -452,7 +454,7 @@ func (m *Master) InstallAPIs(apiResourceConfigSource serverstorage.APIResourceCo
 		}
 		apiGroupInfo, enabled, err := restStorageBuilder.NewRESTStorage(apiResourceConfigSource, restOptionsGetter)
 		if err != nil {
-			return fmt.Errorf("problem initializing API group %q : %v.", groupName, err)
+			return fmt.Errorf("Problem initializing API group %q : %v", groupName, err)
```
Comment (liggitt) [inline pkg/master/master.go]: convention lowercases leading character of errors

**Pull request merged**